Build/Symbols/Symbols.dnn

```diff
@@ -1,6 +1,6 @@
-﻿<dotnetnuke type="Package" version="5.0">
+<dotnetnuke type="Package" version="5.0">
   <packages>
-    <package name="Dnn.PersonaBar.UI_Symbols" type="Library" version="07.01.02">
+    <package name="Dnn.PersonaBar.UI_Symbols" type="Library" version="01.06.01">
```

Comment:
This change will result in an install issue as it is making the version # lower than that of the installed one